src/Compiler/Microsoft.NET.Sdk.Razor.SourceGenerators/Diagnostics/RazorSourceGeneratorResources.resx

```diff
@@ -133,31 +133,31 @@
     <value>TargetPath not defined</value>
   </data>
   <data name="TargetPathNotProvidedMessage" xml:space="preserve">
-    <value>TargetPath not specified for additional file: {0}.</value>
+    <value>TargetPath not specified for additional file: {0}</value>
```

Comment:
Is there a reason this change is in this PR?

Reply:
Hmm, I made that change as it was an analyzer warning, but actually we should just suppress it via config.